woocommerce-gateway-ebanx/gateways/class-wc-ebanx-gateway.php

```diff
@@ -970,6 +970,15 @@ protected function process_response($request, $order)
 				update_post_meta($order->id, "_ebanx_payment_refunds", $refunds);
 				break;
 			case 'UPDATE':
+				$status = array(
+					'CO' => 'Confirmed',
+					'CA' => 'Canceled',
+					'PE' => 'Pending',
+					'OP' => 'Opened'	
+				);
+
+				$order->add_order_note(sprintf(__('EBANX: The payment has been updated to: %s.', 'woocommerce-gateway-ebanx'), $status[$data->payment->status]));
```

Comment:
One thing is the order status and another is the payment status. This merchant specifically there's a problem that I related to you recently and that envolves pay. The notes have had various updated and generate two shipments of the same product and we couldn't identify where the error was. So, adding this new order with our prefix `EBANX` it will be when the payment was updated. Do you agree?